relay/polkadot/Cargo.toml

```diff
@@ -113,9 +115,6 @@ sp-debug-derive = { workspace = true }
 # just for the coretime migration
 polkadot-parachain-primitives = { workspace = true }
 
-# just for use with zombie-bite to test migration
-pallet-sudo = { workspace = true, optional = true }
```

Comment:
@ggwpez this breaks the flow of ahm-dry-run. Is there a fallback? 

https://github.com/paritytech/ahm-dryrun/blob/81c9aa9f526e9568bc08f3a65371e26fcff3cee9/zombie-bite-scripts/helpers.ts#L137-L163

CC @pepoviola @x3c41a

Reply:
So ZN, if possible, should set //Alice as the manager here and then it should wokr again. 

https://github.com/polkadot-fellows/runtimes/blob/22116f7d02c220db4f7187c6967dbd6bf89274cf/pallets/rc-migrator/src/lib.rs#L702-L707

Reply:
I think this stuff happens deep in the ZB engine.